docs/docs/usage.md

```diff
@@ -103,6 +103,43 @@ final SaltyRTC client = new SaltyRTCBuilder()
         .asInitiator();
 ```
 
+## Trusted keys
+
+TODO
+
+## Dynamically determine server connection info
```

Comment:
https://reviewediting.wordpress.com/2012/07/10/titles-sentence-style-or-headline-style/

> It’s also a good style to use if you have many long titles in your document.

I sometimes prefer that to the more complex rules of title capitalization depending on who you ask. This is the AP style:

> Capitalize the first word of every letter except articles, coordinating conjunctions, and prepositions of three letters or fewer. There’s one exception: Any word that is the first word in the headline or the last word should be capitalized, regardless of its part of speech.

And they call that "simple". WTF.

Anyways, feel free to fix the titles if they bother you.